### PR TITLE
Merge delima branch to main

### PR DESCRIPTION
**Pull request by delima-justine**
Please review my code, thank you!

### CODE REVIEW

File: bnpackage/delima.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        print('[2] Goals')
+        print('[3] Comments')
+        print('[4] Exit')
+        user_choice = int(input("Enter your choice: "))
+
+        match user_choice:
+            case 1: #  Basic Info
+                print('\nAge: 19')
+                print('Birthdate: February 24, 2005')
+                print('Currentt School Attended: Polytechnic of the Philippines-Taguig\n')
```
Comment (ynion-mabeamae) [inline bnpackage/delima.py]: Please check your spelling and add University.

**Review by ynion-mabeamae — CHANGES_REQUESTED** [review]
Please review your code before I approve.

File: bnpackage/delima.py (hunk 1 above)
Comment (relente-patriciajoy) [inline bnpackage/delima.py]: You have a missing word to your current school attended.

**Review by relente-patriciajoy — CHANGES_REQUESTED** [review]
Complete the name of your current school attended before I approve it.

File: bnpackage/delima.py (hunk 1 above)
Comment (quiambao-mapatriciaanne) [inline bnpackage/delima.py]: Great job so far! but I noticed that you add extra 't' in your current.

**Review by quiambao-mapatriciaanne — CHANGES_REQUESTED** [review]
Please review your error before I approve it. :>

File: bnpackage/delima.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                print('\nGoal: To have a stable life.\n')
+            case 3: # Comments
+                pass
+                # Ex: print("Comment -Name")
+                # TODO Ynion
+                # TODO Patricia Joy
+                # TODO Patricia Anne
+                # TODO Kath
+            case 4: # Exit
+                print('\nGoodbyye!\n')
```
Comment (citron-kathleen) [inline bnpackage/delima.py]: Please correct your spelling.

**Review by citron-kathleen — CHANGES_REQUESTED** [review]
Please check your code and also, follow the coding guidelines.

**Review by ynion-mabeamae — APPROVED** [review]
Good job!

**Review by citron-kathleen — APPROVED** [review]
Good job!

**Review by quiambao-mapatriciaanne — APPROVED** [review]
Thumbs up!

**Review by relente-patriciajoy — APPROVED** [review]
Well done!

**Pull request merged**